migrations/migrate_add_created_at_to_students.rb

```diff
@@ -0,0 +1,32 @@
+def do_migrate!
+  Classroom::Database.within_each do
+    Classroom::Collection::CourseStudents.add_created_at_field
+  end
+end
+
+module Classroom::Collection::CourseStudents
+  def self.add_created_at_field
+    mongo_collection.find.each do |json|
+      data = json.deep_symbolize_keys
```

Comment:
because is faster. it fetched plain json from DB and I don't need to wrap it. 
Also, if I use `all` I lose the `_id` field and I need it to get the `generated_time` from it